app/models/metric/helper.rb

```diff
@@ -198,4 +198,12 @@ def self.get_time_interval(obj, timestamp)
 
     start_time..timestamp
   end
+
+  def self.latest_metrics(resource_type, resource_ids = nil)
+    metrics = Metric.where(:resource_type => resource_type)
+    metrics = metrics.where(:resource_id => resource_ids) if resource_ids
+    metrics = metrics.order(:resource_id, :timestamp => :desc)
+    metrics = metrics.where('timestamp > ?', 10.minutes.ago.utc)
```

Comment:
@zakiva then it seems to me that this should be a single constant in the code (or a setting?) instead that having `10.minutes.ago.utc` hard-coded, so in case we need to tweak it we change the value in a single place.